Source/Storytellers/Nesi/NesiStoryteller.cs

```diff
@@ -96,42 +73,89 @@ public override IEnumerable<FiringIncident> MakeIntervalIncidents(IIncidentTarge
                     UpdateState();
                 }
 
-                yield return inc;
+                yield return incident;
             }
 
             //return base.MakeIntervalIncidents(target);
         }
 
+        private int GetIncidentCount(IIncidentTarget target)
+        {
+            float acceptFraction = GenerateAcceptFraction(target);
+            return IncidentCycleUtility.IncidentCountThisInterval(
+                target,
+                Find.Storyteller.storytellerComps.IndexOf(this),
+                Props.minDaysPassed,
+                Props.onDays,
+                Props.offDays,
+                Props.minSpacingDays,
+                Props.minIncidents,
+                Props.maxIncidents,
+                acceptFraction
+                );
+        }
+
+        private float GenerateAcceptFraction(IIncidentTarget target)
+        {
+            var acceptFraction = 1f;
+
+            if (Props.acceptFractionByDaysPassedCurve != null)
+            {
+                acceptFraction *= Props.acceptFractionByDaysPassedCurve.Evaluate(GenDate.DaysPassedFloat);
+            }
+
+            if (Props.acceptPercentFactorPerThreatPointsCurve != null)
+            {
+                acceptFraction *= Props.acceptPercentFactorPerThreatPointsCurve.Evaluate(
+                    StorytellerUtility.DefaultThreatPointsNow(target));
+            }
+
+            if (Props.acceptPercentFactorPerProgressScoreCurve != null)
+            {
+                acceptFraction *= Props.acceptPercentFactorPerProgressScoreCurve.Evaluate(
+                    StorytellerUtility.GetProgressScore(target));
+            }
+
+            return acceptFraction;
+        }
+
         private void UpdateState()
         {
+            StorytellerState currentState = StorytellerData.state;
+            int currentDaysPassed = GenDate.DaysPassed;
+            int daysSinceLastStateUpdate = StorytellerData.dayLastUpdated;
```

Comment:
That's a misnomer. It's the last day when it was updated, not how many days it has been since it was updated.